content/en/docs/contribute/new-content/blogs-case-studies.md

```diff
@@ -85,7 +85,7 @@ To submit a blog post, follow these steps:
   - If you are looking for greater coordination on post release dates, coordinating with
     [CNCF marketing](https://www.cncf.io/about/contact/) is a more appropriate choice than submitting a blog post.
   - Sometimes reviews can get backed up. If you feel your review isn't getting the attention it needs,
-    you can reach out to the blog team via [this slack channel](https://kubernetes.slack.com/messages/sig-docs-blog/)
+    you can reach out to the blog team via [`#sig-docs-blog` Slack channel](https://kubernetes.slack.com/messages/sig-docs-blog/)
```

Comment:
nit:
you can reach out to the blog team on the [`#sig-docs-blog` Slack channel](https://kubernetes.slack.com/messages/sig-docs-blog/)